functions/.runtimeconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "authinfo": {
+    "refreshtoken": "1//04Icyb4pe6oDTCgYIARAAGAQSNwF-L9IrRCI89eyvhxltZ4v0-FKOZXOCfhDOp4yco7KnwbMKfdo93sIVB1WmUcWgCjmU4BOB-Hk",
+    "id": "95521899365-41eshv4iv6627iff0gv4pa1cm7c38cuj.apps.googleusercontent.com",
+    "secret": "zRr0l5dr1HTuBN9X2QocSRQl"
+  }
+}
```

Comment:
secrets were pushed to GitHub, please delete and rotate keys @jfeng5529

Reply:
sorry, replaced everything.